doc/diffs.md

```diff
@@ -68,53 +166,184 @@ The "jit-diff list" command has this help message:
 
-Note: you create the `CORE_ROOT` directory "layout" by running the runtest script.
-On Windows, this can be created by running
+You create the `CORE_ROOT` directory "layout" by running the runtest script.
+On Windows, this can be created by running the following in the dotnet/coreclr repo root.
 ```
-    tests\runtest.cmd
+    c:\gh\coreclr> tests\runtest.cmd
 ```
 or
 ```
-    tests\runtest.cmd GenerateLayoutOnly
+    c:\gh\coreclr> tests\runtest.cmd GenerateLayoutOnly
 ```
-in the dotnet/coreclr repo. On non-Windows, consult the test instructions
+On non-Windows, consult the test instructions
 [here](https://github.com/dotnet/coreclr/blob/master/Documentation/building/unix-test-instructions.md).
 Note that you can pass `--testDir=NONE` to runtest.sh to get the
 same effect as passing `GenerateLayoutOnly` to runtest.cmd on Windows.
 
-## jit-analyze
+This command will generate the diffs into the specified output directory. It will automatically create a "tag"
+subdirectory (if the `--tag` switch isn't specified to override the default name), and within that tag
+directory will be "base" and "diff" directories, containing the diffs. The default tag directory looks like
+`dasmset_12`, with a unique number for every run of jit-diff.
 
-The jit-analyze tool understands the format of the `*.dasm` files in a diff and can extract
-this data to produce a summary and/or dump the info to a data file (in CSV or JSON format).
-The common usage of the tool is to extract interesting diffs, if any, from a diff run as
-part of the development progresses.
+jit-analyze will be run to compare the generated output.
 
-This tool can be invoked automatically after generating diffs using the `-a` option to `jit-diff diff`.
+You can also run a recursive textual comparison tool like windiff on Windows to visually compare
+the diffs, e.g.:
+```
+    c:\gh\coreclr> windiff c:\diffs\dasmset_12\base c:\diffs\dasmset_12\diff
+```
 
-Sample help command line:
+## Simplified jit-diff usage: defaults
+
+As seen above, specifying all required information can be quite verbose. jit-diff can automatically determine
+most of the arguments using computed defaults. The above diff can be accomplished using simply:
+```
+    c:\gh\coreclr> jit-diff diff --base_root e:\gh\coreclr2
+```
+
+The defaults are:
+* If jit-diff is invoked with the current directory within the dotnet/coreclr repo, then the root of
+  this repo serves to find the diff compiler and `CORE_ROOT` directory. (Note that we have no reasonable
+  default for determining what the baseline toolset or repo is. This is specified with the `--base_root` argument.)
+* The default output directory is `<repo_root>\diffs` (in this case, c:\gh\coreclr\diffs).
+* The default diff architecture is x64. If this isn't found, x86 is tried.
+* The default diff build flavor is checked. If this isn't found, debug is tried.
+* By default, diffs are done using System.Private.CoreLib.dll. (That is, `--corelib` is the default.)
```

Comment:
You can always just pass everything explicitly, as today. Is that not sufficient?

I can also see having a different set of build flavor defaults for core_root/test_root and the JIT directories, and not trying to get them all to match when computing defaults. I'm leery of adding additional `--build` arguments, though: it seems like you should just pass the path explicitly if you don't like the default.

It seems like you are suggesting:
`--core_root` and `--test_root` should default to release, then fall back to checked or debug. (Or not fall back at all? I can't agree with that option, since my goal is to have full defaults for everything.)
`--diff` and `--base` should default to debug, then fall back to checked (or vice-versa, as I currently have it?)

I normally build a checked JIT before doing diffs, as it's faster.